main.go

```diff
@@ -60,9 +59,8 @@ var (
 func main() {
 	// listen on port 8080 for any event
 	logger := keptncommon.NewLogger("", "", "prometheus-service")
+
 	logger.Debug("Starting server for receiving events on exposed port 8080")
-	http.HandleFunc("/", Handler)
-	go http.ListenAndServe(":8080", nil)
```

Comment:
Could you please explain why you removed this?

Reply:
Please re-add this again :) It's needed AFAIK

Reply:
I think that's not needed since it already starts something on 8080 with https://github.com/keptn-contrib/prometheus-service/blob/master/main.go#L88 or https://github.com/keptn-contrib/prometheus-service/blob/master/main.go#L84. On my side it was not possible than to have the pod starting properly.

Reply:
There is actually a huge problem with this (see #195), but that's a separate issue ;)